src/routes/forgot-password/components/ForgotPasswordPage.js

```diff
@@ -0,0 +1,86 @@
+const { useState } = require('react');
+const T = require('prop-types');
+const { default: Styled } = require('styled-components');
+const { default: Typography } = require('@mui/material/Typography');
+const { default: TextField } = require('@mui/material/TextField');
+const { default: Button } = require('@mui/material/Button');
+const { default: Box } = require('@mui/material/Box');
+
+const internals = {};
+
+module.exports = function ForgotPasswordPage({ onPressForgotPassword }) {
+
+    const [email, setEmail] = useState('');
+    const [isSubmitting, setIsSubmitting] = useState(false);
+    const [didEmailSend, setDidEmailSend] = useState(false);
+
+    const disableSubmit = () => {
+
+        return !email;
+    };
+
+    const handleSubmit = async (ev) => {
+
+        ev.preventDefault();
+        const accountInfo = formatFields();
```

Comment:
No difference; just adding an extra abstraction via a little helper function. I agree it looks silly when there's only one, simple field, but it can make things more readable when there's lots of state and mutations that you don't want to interfere with submission logic.

Reply:
That makes sense 🙏 Thank you!